test/suite/test_tiered07.py

```diff
@@ -147,7 +160,11 @@ def test_tiered(self):
 
         # Create new table with new name.
         self.pr('create new table')
-        self.session.create(self.newuri, 'key_format=S')
+        self.session.create(self.newuri, 'key_format=S,value_format=S,tiered_storage=(name=none)')
+
+        # Check that the underlying files are not removed when configured in drop.
+        self.session.drop(self.newuri, 'remove_files=false')
```

Comment:
It's good that we test removal of non-tiered tables, but we also should test tiered tables and check if the underlying wtobj file is deleted/not deleted.

Reply:
My bad, I meant to create a tiered table.

Reply:
Currently, the wtobj files are not being removed, regardless of whether we use the remove_files configuration or not.

Reply:
Good catch! I've created a ticket for the issue: WT-9003